regression/scripts/aerodynamics_super/aerodynamics_super.py

```diff
@@ -1,502 +0,0 @@
-# test_aerodynamics
```

Comment:
was this script even being used before in the regression? Did this actually have some coverage?

Reply:
No it wasn't. It did not have any coverage